src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double fistLeg;

```

Comment:
There is a typo in the variable name `fistLeg`. It should be `firstLeg` to accurately describe the first leg of the triangle.